src/components/PortfolioAccordions.vue

```diff
@@ -54,7 +59,7 @@
           aria-expanded="true"
           aria-controls="m-a2"
         >
-          Chart Gallery
+          Chart Gallery <span id="text-info"> Click to see in the wild</span>
```

Comment:
Just a note - because this is blue, I thought this was a hyperlink at first. Maybe clarify to say - click on each card to see the viz in the wild? Or make the font black/grey?